### PR TITLE
SAMZA-2625: Remove PrintGCDateStamp (Deprecated in JDK 11)

### PR DESCRIPTION
**Pull request by bringhurst**

### CODE REVIEW

**Comment by mynameborat** [comment]
@bringhurst Can you please fill out the description according [SEP-25: PR Title And Description Guidelines](https://cwiki.apache.org/confluence/display/SAMZA/SEP-25%3A+PR+Title+And+Description+Guidelines) before I merge the PR?

File: samza-shell/src/main/bash/run-class.sh [hunk 1]
```diff
@@ -154,9 +154,6 @@ fi
 # Check if a max-heap size is specified. If not - set a 768M heap
 [[ $JAVA_OPTS != *-Xmx* ]] && JAVA_OPTS="$JAVA_OPTS -Xmx768M"
 
-# Check if the GC related flags are specified. If not - add the respective flags to JVM_OPTS.
-[[ $JAVA_OPTS != *PrintGCDateStamps* && $JAVA_OPTS != *-Xloggc* ]] && JAVA_OPTS="$JAVA_OPTS -XX:+PrintGCDateStamps -Xloggc:$SAMZA_LOG_DIR/gc.log"
```
Comment (mynameborat) [inline samza-shell/src/main/bash/run-class.sh:158]: Does default GC flag for `loggc` configured to generate `gc.log` in the right directory?

Comment (kw2542) [inline samza-shell/src/main/bash/run-class.sh:158]: Can you double check whether we need to keep `-Xloggc:$SAMZA_LOG_DIR/gc.log` ?

**Comment by bringhurst** [comment]
I'm going to close this for now -- I wasn't thinking that this affected the log path. I'll reopen or create another PR when it's ready.

**Pull request closed without merging**